statics/js/components/topology.js

```diff
@@ -174,12 +174,16 @@ var TopologyComponent = {
   ',
 
   data: function() {
+    var filter = "";
+    if (typeof(Storage) !== "undefined") {
+      filter = localStorage.getItem("defaultFilter");
+    }
```

Comment:
Instead of using localStorage directly, we should look at vuex alternatives:
- https://github.com/robinvdvleuten/vuex-persistedstate
- https://github.com/crossjs/vuex-localstorage

So that we can have 2 vues stores, one persistent, one temporary (the one we already have).

In this case, if the topologyFilter is stored in a persistent vuex store, we can react on it in all components.

Reply:
In fact with https://github.com/robinvdvleuten/vuex-persistedstate it is possible to specify which keys from the store are persisted, so only one store is required.

Reply:
I think we can start the current approach if masco is fine with it, then we will improve this.

Reply:
I am thinking these values are no need to available for all the vue components and not a big fan of using more external libs unless it is really needed.